app/assets/stylesheets/local/work_show.scss

```diff
@@ -33,5 +33,15 @@
     max-width: $max-readable-width;
   }
 
+  .show-sub-head-body {
+    border-top: 3px solid #ddd;
+    padding: 5px;
+  }
+  .citation {
+      line-height: 1.5 !important;
+      word-break: break-word;
+  }
+
```

Comment:
i wanna look at the CSS and see if there's a way to make it more consistent with existing CSS, perhaps using existing SASS variables from bootstrap, or other styles we already have. But there might not be. It's confusing to get a handle on the CSS, for me too. So just want to look at it a bit.

Reply:
Oh yeah absolutely

Reply:
Thanks for the CSS!